openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/CPDCallback.java

```diff
@@ -141,8 +143,8 @@ private void visitCpdNode(JPNode node) {
       TextRange range = file.newRange(node.getLine(), node.getColumn(), node.getEndLine(), node.getEndColumn());
       cpdTokens.addToken(range, str);
     } catch (IllegalArgumentException uncaught) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1166)